src/config.rs

```diff
@@ -154,19 +154,19 @@ configuration_option_enum! { ReportTactic:
 }
 
 configuration_option_enum! { WriteMode:
-    // Backsup the original file and overwrites the orignal.
+    // Backups the original file and overwrites the orignal.
```

Comment:
I think "backsup" is correct here, although it should probably be "backs up" or "backs-up"